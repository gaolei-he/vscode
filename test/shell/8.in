14
/cab
/cc/cd
/ab
/bd/bc/acd
/d/c/c
/da/c
/cc
/aa/baa/dad
/a/ad/cd
/aa/cb/dcc
/c
/dad/adc
/cbc/c/b
/dac/ab/ab
7
ls
pwd
ls
ls
cd .
pwd
pwd
