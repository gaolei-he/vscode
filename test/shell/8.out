a
aa
ab
bd
c
cab
cbc
cc
d
da
dac
dad
/
a
aa
ab
bd
c
cab
cbc
cc
d
da
dac
dad
a
aa
ab
bd
c
cab
cbc
cc
d
da
dac
dad
/
/
